### PR TITLE
[Tracing][PHP] Document IP collection in PHP (and .NET)

### PR DESCRIPTION
**Pull request by pierotibou**
### What does this PR do?
States that we are reintroduing IP Collection, disabled by default, in the php tracer.

### Motivation
Feature asked by clients. Will be merged when https://github.com/DataDog/dd-trace-php/pull/1895 is released

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_collection/library_config/dotnet-core.md [hunk 1]
```diff
@@ -176,6 +176,12 @@ Added in version 1.18.3. Response header support and entries without tag names a
 Added in version `2.19.0`.<br>
 **Default**: `false`<br>
 
+`DD_TRACE_CLIENT_IP_HEADER`
+: The IP header to be used for client IP collection, for example: `x-forwarded-for`. <br>
+Added in version `2.19.0`.<br>
+**Default**: We parse all the followings. `"x-forwarded-for", "x-real-ip", "client-ip", "x-forwarded", "x-cluster-client-ip", "forwarded-for", "forwarded", "via", "true-client-ip"`. If several are present, none will be reported.<br>
```
Comment (jtappa) [inline content/en/tracing/trace_collection/library_config/dotnet-core.md:182]:
```suggestion
**Default**: Datadog parses the following: `"x-forwarded-for", "x-real-ip", "client-ip", "x-forwarded", "x-cluster-client-ip", "forwarded-for", "forwarded", "via", "true-client-ip"`. If several are present, none will be reported.<br>
```

File: content/en/tracing/trace_collection/library_config/dotnet-framework.md [hunk 2]
```diff
@@ -196,6 +196,11 @@ Added in version 1.18.3. Response header support and entries without tag names a
 Added in version `2.19.0`.<br>
 **Default**: `false`<br>
 
+`DD_TRACE_CLIENT_IP_HEADER`
+: The IP header to be used for client IP collection, for example: `x-forwarded-for`. <br>
+Added in version `2.19.0`.<br>
+**Default**: We parse all the followings. `"x-forwarded-for", "x-real-ip", "client-ip", "x-forwarded", "x-cluster-client-ip", "forwarded-for", "forwarded", "via", "true-client-ip"`. If several are present, none will be reported.<br>
```
Comment (jtappa) [inline content/en/tracing/trace_collection/library_config/dotnet-framework.md:202]:
```suggestion
**Default**: Datadog parses the following: `"x-forwarded-for", "x-real-ip", "client-ip", "x-forwarded", "x-cluster-client-ip", "forwarded-for", "forwarded", "via", "true-client-ip"`. If several are present, none will be reported.<br>
```

File: content/en/tracing/trace_collection/library_config/php.md [hunk 3]
```diff
@@ -298,14 +298,19 @@ A comma-separated list of query parameters to be collected as part of the URL. S
 **Default**: `*`<br>
 A comma-separated list of query parameters to be collected as part of the resource URI. Set to empty to prevent collecting any parameters, or `*` to collect all parameters. Added in version `0.74.0`.
 
+`DD_TRACE_CLIENT_IP_ENABLED`
+: **INI**: `datadog.trace.client_ip_enabled`<br>
+**Default**: `false`<br>
+Enables IP Collection client side. Added in version `0.84.0`.
```
Comment (jtappa) [inline content/en/tracing/trace_collection/library_config/php.md:304]:
```suggestion
Enables IP collection client side. Added in version `0.84.0`.
```

**Review by jtappa — CHANGES_REQUESTED** [review]

**Comment by pierotibou** [comment]
Thanks a lot @jtappa for the review. I've applied all your changes

**Review by jtappa — APPROVED** [review]

**Comment by jtappa** [comment]
Looks like the runners were full so the build failed, re-running in gitlab

**Pull request merged**